server/src/graphql-types/User.ts

```diff
@@ -21,25 +21,7 @@ export class UserWithInstanceRole extends User {
 }
 
 @ObjectType()
-export class UserWithRelations extends User {
-  @Field(() => [Chapter])
-  created_chapters: Chapter[];
-
-  @Field(() => [UserBan])
-  banned_chapters: UserBan[];
-
-  @Field(() => [ChapterUser])
-  chapter_users: ChapterUser[];
-
-  @Field(() => [InstanceRole])
-  instance_role: InstanceRole[];
-
-  @Field(() => [EventUser])
-  event_users: EventUser[];
-}
-
-@ObjectType()
-export class UserInformation extends User {
+export class UserInformation extends UserWithInstanceRole {
```

Comment:
this is needed, so userinformation query have admined_chapters

But it won't be needed after refactoring so it should be reverted in the future PR